test/alternator/test_return_consumedcapacity.py

```diff
@@ -0,0 +1,83 @@
+# Copyright 2024-present ScyllaDB
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+# Tests for the ReturnConsumedCapacity header
+
+import pytest
+from botocore.exceptions import ClientError
+from decimal import Decimal
+from test.alternator.util import random_string, random_bytes
+
+# A basic test that gets an item from a table with and without consistency
+KB = 1024
+
+def test_basic_get(test_table):
+    p = random_string()
+    c = random_string()
+    val = random_string()
+    val2 = random_string()
+    test_table.put_item(Item={'p': p, 'c': c, 'attribute': val, 'another': val2})
+    response = test_table.get_item(Key={'p': p, 'c': c}, ConsistentRead=True, ReturnConsumedCapacity='TOTAL')
+    item = response['Item']
+    assert item['p'] == p
+    assert item['c'] == c
+    assert item['attribute'] == val
+    assert item['another'] == val2
```

Comment:
Nitpick: I know I wrote some tests l like that a long time ago, but today I prefer the shorter and nicer syntax
```python
assert item == {'p': p, 'c': c, 'attribute': val, 'another': val2}
```